CHANGES/8847.feature

```diff
@@ -0,0 +1,2 @@
+Added support for specifying a minor version for `pulpcore_version`. A single version of pulp_installer can be used to
+install all bug fix releases within a minor release of pulpcore.
```

Comment:
Some users have been setting pulp_version.

We could reach out to them individually (reply to mailing list convos), or add a removal notice.

Reply:
I'll add a removal notice.